Tests/Svg.UnitTests/ImageComparisonTest.cs

```diff
@@ -44,6 +44,9 @@ public void CompareSvgImageWithReference(ImageTestDataSource.TestData testData)
         [TestCaseSource(typeof(ImageTestDataSource), "PassingTests")]
         public void CompareSvgImageWithReference(ImageTestDataSource.TestData testData)
         {
+            // W3C Test Suites use external references to local fonts
+            SvgDocument.ResolveExternalResources = true;

```

Comment:
It seems better to review the settings here as well.

https://github.com/svg-net/SVG/blob/062dc7c91cadf5607cceca1103f38628c0e065f8/Tests/Svg.UnitTests/ImageComparisonTest.cs#L94

Reply:
Well spotted. I've removed the assignment in `CompareSvgImageWithReference` and moved the assignment in `CompareSvgImageWithReferenceImpl` to the top of the method instead.